doc/sources/guide/packaging-osx.rst

```diff
@@ -1,82 +1,192 @@
 Creating packages for OS X
 ==========================
 
-Packaging your application for the OS X 10.6 platform can only be done inside OS X.
-
-The package will only work for the 64 bit OS X. We no longer support 32 bit OS X platforms.
-
 .. note::
-    Currently, packages for OS X can only be generated with Python 2.7. Python 3.3+ support is on the way...
+    - Packaging your application for the OS X platform can only be done inside OS X.
 
-.. _osx_requirements:
 
-Official Packaging method
--------------------------
+Using kivy-sdk-packager
+-----------------------
+.. note::
+    - This is currently the preferred way to package an app if you are new to Kivy
+      and/or you know that you do not need newer or more libraries than provided by the Kivy.app
+    - The package will only work for the 64 bit OS X. We no longer support 32 bit OS X platforms.
+    - Currently, packages built using this method can only be generated with Python 2.7.
+      Python 3.3+ support is on the way...
 
```

Comment:
We simply don not support older OsX 32bit, not just for packaging but for kivy itself too. has nothing to do with current packaging method.

The limitation to python2.7 was for pyinstaller that didn't include py3 support instill recently not for this packaging method.